core/Piranha.Manager.Core/Menu.cs

```diff
@@ -18,19 +18,21 @@ public static class Menu
         /// <summary>
         /// The basic manager menu.
         /// </summary>
-        public static MenuItemList Items = new MenuItemList
+        public static MenuItemList Items { get; } = new MenuItemList
         {
             new MenuItem
             {
                 InternalId = "Content", Name = "Content", Css = "fas fa-pencil-alt", Items = new MenuItemList
                 {
                     new MenuItem
                     {
-                        InternalId = "Pages", Name = "Pages", Route = "~/manager/pages", Policy = Permission.Pages, Css = "fas fa-copy"
+                        InternalId = "Pages", Name = "Pages", Route = "~/manager/pages", Policy = Permission.Pages,
+                        Css = "fas fa-copy"

```

Comment:
I would prefer one line each property instead put only the last line property in a new line because of reaching the prefered max line length

Reply:
right, fixed

Reply:
right, fixed :)